README.md

```diff
@@ -1,2 +1,218 @@
-# hclust
-Agglomerative hierarchical clustering in JavaScript
+### hclust
+[Agglomerative hierarchical clustering](https://en.wikipedia.org/wiki/Hierarchical_clustering) in JavaScript
+
+Adapted from [hcluster.js](https://github.com/cmpolis/hcluster.js) by [@ChrisPolis](https://twitter.com/chrispolis)
+
+---
+
+### Usage
+
+```html
+<script src="hclust.min.js"></script>
+<script>
+  hclust.clusterData(...);
+  hclust.euclideanDistance(...);
+  hclust.avgDistance(...);
+  hclust.minDistance(...);
+  hclust.maxDistance(...);
+</script>
+```
+
+or 
+
+```javascript
+import { clusterData } from 'hclust';
+import { euclideanDistance } from 'hclust';
+import { avgDistance } from 'hclust';
+import { minDistance } from 'hclust';
+import { maxDistance } from 'hclust';
+```
```

Comment:
Should we make this an npm package? Either way, I'll add a line on how to install this, which will either be `npm install hclust` or `npm install git+https://github.com/greenelab/hclust.git`

Reply:
I believe it is stable. The question is more, will we maintain this over time. And also, have my changes (compared to hcluster.js) made it impractical for anyone else to use but us (it works in our case, but maybe other people do need stuff from the input `data` to be duplicated in the results).